frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/grid_page.dart

```diff
@@ -83,7 +83,7 @@ class _GridPageState extends State<GridPage> {
             loading: (_) =>
                 const Center(child: CircularProgressIndicator.adaptive()),
             finish: (result) => result.successOrFail.fold(
-              (_) => const GridShortcuts(child: FlowyGrid()),
+              (_) => const  GridShortcuts(child: FlowyGrid() ),
```

Comment:
```suggestion
              (_) => const GridShortcuts(child: FlowyGrid()),
```

remove whitespace changes please :)